genesis/README.md

```diff
@@ -0,0 +1,32 @@
+# Genesis
+
+The Genesis package converts formatted JSON files into the genesis of the Primary Network. For the simplest example, see the [Local Genesis](./genesis_local.json) JSON file.
```

Comment:
```suggestion
The Genesis package converts formatted JSON files into the genesis of the Primary Network. For the simplest example, see the [Local Genesis](https://github.com/ava-labs/avalanchego/blob/master/genesis/genesis_local.json) JSON file.
```
This suggestion is only necessary if we also want to import this page into the docs. imo we should cc: @ashucoder9